src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -376,6 +376,49 @@ public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestP
             }
         }
 
+        public virtual async Task<IList<Commit>> GetPullRequestCommitsAsync(string pullRequestUrl)
+        {
+            (string owner, string repo, int id) = ParsePullRequestUri(pullRequestUrl);
+            Octokit.PullRequest pr = await Client.PullRequest.Get(owner, repo, id);
+            var pullRequestCommits = await Client.PullRequest.Commits(owner, repo, pr.Number);
+            IList<Commit> commits = new List<Commit>(pullRequestCommits.Count);
+            for (int i = 0; i < pullRequestCommits.Count; i++)
+            {
+                commits.Add(new Commit(pullRequestCommits[i].Commit.Author.Name,
+                    pullRequestCommits[i].Sha,
+                    pullRequestCommits[i].Commit.Message));
+            }
+            return commits;
+
+        }
+
+        /// <summary>
+        ///     Merge a dependency update pull request
+        /// </summary>
+        /// <param name="pullRequestUrl">Uri of pull request to merge</param>
+        /// <param name="parameters">Settings for merge</param>
+        /// <param name="commitToMerge">Actual commit message</param>
```

Comment:
The name of the variable and this comment don't really mesh.  Is it a commit message or commitsToMerge? Though "commitsToMerge" is a weird name for string parameter...

Reply:
changed it to mergeCommit

Reply:
I think the question is still relevant. The name implies it's a commit, but it's actually a commit message.